pkg/machinehandler/machinehandler_test.go

```diff
@@ -67,10 +68,22 @@ func (f fakeMachineRoundTripper) RoundTrip(req *http.Request) (*http.Response, e
 				  }
 			]
 		}`
+	} else {
```

Comment:
You should use the path here, not just assume all remaining requests hit this point, the path should be `/apis/<group>/<version>` so you'll need to account for both versions tested in this test suite